package.json

```diff
@@ -31,6 +33,8 @@
     "node-sass": "^4.11.0",
     "node-wp-i18n": "~1.2.1",
     "prettier": "npm:wp-prettier@^2.2.1-beta-1",
+    "react": "17.0.1",
+    "react-dom": "17.0.1",
```

Comment:
It doesn't look like React 17 is part of WP core. WP 5.8 (the latest version as of now) uses React 16.13.1. In order to avoid using newer APIs that might not be part of React 16, what do you think about ensuring that we declare the same version as the one used in WP 5.8?

Reply:
👍 going with `16.13.1`. Using the highest available 16.x version of `@types/react` and `@types/react-dom`